build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id("org.springframework.boot") version "2.5.2"
```

Comment:
Defines versions of dependencies used in the Spring Boot project. https://docs.spring.io/spring-boot/docs/current/reference/html/dependency-versions.html